src/main/java/backend/IssueMetadata.java

```diff
@@ -12,18 +12,26 @@
     private final List<TurboIssueEvent> events;
     private final List<Comment> comments;
     private final LocalDateTime nonSelfUpdatedAt;
+    private final LocalDateTime selfUpdatedAt;
     private final int nonSelfCommentCount;
+    private final int selfCommentCount;
 
     // If isUpdated is true, nonSelfUpdatedAt will be used to sort/filter instead of updatedAt in TurboIssue
     private final boolean isUpdated;
+    private final boolean isUpdatedBySelf;
+    private final boolean isUpdatedByOthers;
```

Comment:
Is there a difference between `isUpdatedByOthers` and `isUpdated`? See the comment above.